Service/EventLogCleanup.php

```diff
@@ -107,9 +124,16 @@ public function deleteEventLogEntries(int $daysOld, ?int $campaignId, bool $dryR
                     continue;
                 }
 
-                $sql                     = 'SELECT * FROM '.str_replace(self::PREFIX, $this->dbPrefix, $this->queries[$operation]);
-                $statement               = $this->connection->executeQuery($sql, $this->params[$operation], $this->types[$operation]);
-                $result[$operation]      = $statement->rowCount();
+                if (true === $operations['email_stats_tokens']) {
```

Comment:
Use constant key `self::EMAIL_STATS_TOKENS`

Reply:
Thanks @Moongazer for the recommendation to better use constans if we have them - changed this not harmful part in the better practise part with my current commit.